### PR TITLE
Jordan slim

### PR DESCRIPTION
**Pull request by JordanSlim1**

### CODE REVIEW

**Comment by JordanSlim1** [comment]
Had to implement a contact form on contact page.

File: contact.css [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    grid-area: main;
+    font-size: 1rem;
+    color: black;
+    width: 100%;
+    height: auto;
+    font-size: auto;
+    text-align: center;
+}
+
+@media screen and (max-width: 1200px) {
```
Comment (ayunas) [inline contact.css:32]: do you have a screen size bigger than 1200px?  is this for a tablet view?

File: gallery.css [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+}
+
+.photo3 img {
+    margin: auto;
+    height: auto;
+    width: 40%;
+}
+
+.photo4 {
+    background-color: khaki;
```
Comment (ayunas) [inline gallery.css:56]: nice color

File: index.html [hunk 3]
```diff
@@ -1,11 +1,43 @@
...
+        <div class="head">
+            <hgroup>
+                <h1>Hello Everyone!</h1>
+                <h2>I am Jordan Slim</h2>
+            </hgroup>
+        </div>
+        <div class="photo">
+            <img src="Pictures/personalphoto.JPG">
+        </div>
+        <div class="main"> I am 23 years old. I graduated with a computer science degree at UCA.
```
Comment (ayunas) [inline index.html:22]: consider using semantic tags here,  `<main>` for example

**Review by ayunas — APPROVED** [review]
Thanks for your effort Jordan.  

I would focus on design.  Your nav bar changes on different pages, better to keep the nav bar consistent from page to page.  Also look into some color combinations from a UX perspective.  
The colors you chose are clashing too much with each other, which affects User Experience.  


check this out for UX color theory
[](https://uxplanet.org/color-theory-brief-guide-for-designers-76e11c57eaa
)

**Pull request merged**